Console/Command/GraphShell.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-App::uses('AppShell', 'Console\Command');
+App::uses('AppShell', 'Console/Command');
```

Comment:
This isn't needed here.

The `App` class takes care of the DIRECTORY_SEPARATOR.
https://github.com/cakephp/cakephp/blob/2.6.0/lib/Cake/Core/App.php#L521
https://github.com/cakephp/cakephp/blob/2.6.0/lib/Cake/Core/App.php#L534-L572